packages/faucet/src/database-helper.ts

```diff
@@ -26,11 +26,19 @@ export enum RequestType {
   Invite = 'Invite',
 }
 
+enum MobileOS {
+  'andriod' = 'android',
+  ios = 'ios',
+}
+
 export interface RequestRecord {
   beneficiary: Address
   status: RequestStatus
-  txHash?: string
```

Comment:
should we remove this from database-rules too?

Reply:
if i'm understanding this right, yes and we should add `goldTxHash` `dollarTxHash` and `escrowTxHash`

those three are already set on the `ref`

Reply:
yes